drafts/swift/redux-in-swift-pt-1.md

```diff
@@ -0,0 +1,280 @@
+# Redux in Swift Part 1: Introduction and Example within 70 Lines!
+
+In this blog, I will introduce you to the concept of Redux architecture briefly, implement and explain it in Swift bit by bit with minimum changes each time, while satisfying Redux's requirements and making it compilable at the same time.
+
+## Redux 
+
+> Redux architecture revolves around a **strict unidirectional data flow**.
+
+![](https://www.dropbox.com/s/51evjhxw35t59x2/Redux.png?raw=1)
+
+[Redux Doc: Data Flow](http://redux.js.org/docs/basics/DataFlow.html)
+
+### WAT?
+
+In essense, it's an idea which trys to
+
+- seperate/isolate where changes can happen to the "State" of an application
+- limits the flow of the data in an application
+
+### What Composes Redux?
+
+Five types (of things) composes Redux:
```

Comment:
This is an artefact of the English language: Redux is composed _of_ some things, rather than those things _composing_ Redux.*

_*Actually, both are correct, but the way you've written it above is a bit more academic and less easy to parse for an average reader, I'd say._